qadence2_platforms/backends/pyqtorch/functions.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+from typing import cast
+
+import pyqtorch as pyq
+from pyqtorch.hamiltonians import Observable
+from pyqtorch.primitives import Primitive
+
+from qadence2_platforms.backends.pyqtorch.utils import InputType, Support
+
+
+def _get_op(op: InputType) -> Primitive:
+    if op.is_symbol is True:
+        return getattr(pyq, op.head.upper(), None)
+
+    if op.is_quantum_operator is True:
+        op_args_item: InputType = op.args[0].args
+        return getattr(pyq, op_args_item[0].upper(), None)
+
+    op_args: str = cast(str, op.args[0])
+    return getattr(pyq, op_args.upper(), None)
```

Comment:
Isn't there a type mismatch here ? Unless `None` is a sensical `Primitive` type ?